master/getting-started/openstack/upgrade/convert.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Converting your calicoctl manifests
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/openstack/upgrade/convert'
```

Comment:
I don't believe there's a link from the intro page to here.  Is that needed?  If not, I guess let's delete this page.

Reply:
Added.